HIP-xxxx.md

```diff
@@ -0,0 +1,163 @@
+# HIP-xxxx : Standard for TXT records
+
+```
+Number:  HIP-xxxx
+Title:   Standard for TXT records
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal for TXT record standards to provide on-chain data for a domain.
+
+## Motivation
+
+#### Domain Parking
+
+There's currently no way to know if a domain is for sale by the owner or not.
+A domain seller has no reliable way to provide that information.
+
+#### Identity / Web3 Profile
+
+Websites using Sign in with Handshake can use and render the TXT record data, e.g. as profile information. The domain owner is still in control of the data.
+
+## TXT Standards Definition
+
+For the above to work we need standards so any website can properly render the TXT records. The records should be flexible, short and easy to understand. Since it's impossible to validate the data via HSD. Therefore TXT records should be provided mostly without any prefix like "mailto:", "tel:" or "https://www.twitter.com/".
+
+The goal is to let the app or website decide how to render the information. It also decreases the data saved on-chain.
+
+#### Domain Parking
+
+The parking TXT entry indicates that the domain is for sale.
+
+```
+TXT parking=[String]
+```
+
+#### Contact or Profile Information
+
+```
+TXT email=[Email]
+TXT phone=[Tel]
+```
+
+```
+TXT discord=[String]
+TXT facebook=[String]
+TXT github=[String]
+TXT instagram=[String]
+TXT linkedin=[String]
+TXT telegram=[String]
+TXT tiktok=[String]
+TXT twitter=[String]
+TXT weixin=[String]
+TXT whatsapp=[Tel]
+TXT youtube=[String]
+```
+
+#### Avatar
+
+```
+TXT avatar=[ URL | skylink | ipfsLink ]
+```
+
+#### Domain Caption
+
+A caption is especially useful for identities or as a sales pitch. It can also be used to add more context to a domain (e.g. NFT's).
+
+```
+TXT caption=[String]
+```
+
+#### Payment / Wallet Alias
+
+Although [HIP-0002](https://github.com/handshake-org/HIPs/blob/master/HIP-0002.md) is a much better approach, TXT entries could also be used to provide static wallet information.
+
+```
+TXT wallet=[Chain]:[Address]
+```
```

Comment:
I have mixed feelings about this section. I struggle to see a good benefit here.
It's also a bit limited: what if the seller accept multiples chains?

Reply:
The benefits are (in comparison to HIP-0002): 
1. You don't need to host any files
2. It's easy to maintain

The downside is: The wallet addresses are kind of static

If you wanted to support multiple chains my initial thought was
```
TXT wallet=hns:hs1qnkr3cwf9ldp7qpq2csuuyxf0znj0lwhmawhlal
TXT wallet=btc:12dRagNcdxK39288NjcDV4GX7rMsKCGn6B
TXT wallet=eth:0x28346f1ec065eea239152213373bb58b1c9fc93b
```

But after reading RFC 1464 I think it's better to do 

```
TXT wallet hns=hs1qnkr3cwf9ldp7qpq2csuuyxf0znj0lwhmawhlal
TXT wallet btc=12dRagNcdxK39288NjcDV4GX7rMsKCGn6B
TXT wallet eth=0x28346f1ec065eea239152213373bb58b1c9fc93b
```